src/page-objects/administration/PaymentDetail.ts

```diff
@@ -0,0 +1,26 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class PaymentDetail implements PageObject {
+
+    public readonly header: Locator;
+    public readonly nameField: Locator;
+    public readonly availabilityRuleField: Locator;
+    public readonly availabilityRuleListItem: Locator;
+
+
+    constructor(public readonly page: Page) {
+        this.header = page.locator('.smart-bar__header');
+        this.nameField = page.locator('.sw-settings-payment-detail__field-name').getByLabel('Name');
```

Comment:
I double checked. The problem here why you get 2 results with getByLabel('Name'), is getByLabel checks where a label is present which contains "name" ...in your case of course Name field and technical name field because the label of technical name field also called "technical _name_" 

What you could do here to use the locator with "exact: true"

page.getByLabel('Name', { exact: true });